src/pages/Filtering.js

```diff
@@ -1,41 +1,19 @@
-import React from "react";
 import axios from "axios";
 import { useQuery } from "react-query";
-import Layout from "../components/Layout";
 
-const Filtering = () => {
-  const { isLoading, data } = useQuery("posts", () => {
-    return axios.get("https://jsonplaceholder.typicode.com/posts?_limit=10");
-  });
-
-  if (isLoading) {
-    return <h2>Loading....</h2>;
-  }
+export default function App() {
+  const { isLoading, data, error } = useQuery(["posts"], () =>
+    axios("https://jsonplaceholder.typicode.com/posts")
+  );
+  if (error) return <h2>Error </h2>;
+  if (isLoading) return <h2> isLoading </h2>;
   console.log(data);
+  // return <div>
   return (
-    <>
-      <Layout />
-      <h2>Results</h2>
-      {data?.data.map((post) => {
-        return <div key={post.title}>{post.title}</div>;
-      })}
-    </>
+    <div>
+      {data.data.map((data) => (
+        <div>{data.title}</div>
+      ))}
```

Comment:
Javascript is pretty forgiving when it comes to variable names and scoping, so this probably works just fine.  However, it reads very weird and is confusing.

Specifically the use of the term `data` for the outer object, combined with the term `data` for the inner object _and_ `data` again for the individual entries.  At a minimum, I would expect a singular word here as the value for each iteration of the `map` instead of the plural word `data`.  I've used the terms `entry` or `item` in the past; I've even used `datum` before.  However, each of these generic singular terms should be treated as a fallback option; it's far better to use a better term in place of the upstream word `data`.

What _are_ these things?  Are they menu items?  Could you send back `menuItems` from the API and then call each entry in the array a `menuItem`?


```suggestion
      {data.menuItems.map((menuItem) => (
        <div>{menuItem.title}</div>
      ))}
```